src/py21cmfast/inputs.py

```diff
@@ -1054,6 +1055,13 @@ class AstroParams(StructWithDefaults):
     UPPER_STELLAR_TURNOVER_INDEX:
         The power-law index associated with the optional upper mass power-law of the stellar-halo mass relation
         (see FlagOptions.USE_UPPER_STELLAR_TURNOVER)
+    SIGMA_LX: float, optional
+        Lognormal scatter (dex) of the Xray luminosity relation.
+        Uniform across all masses and redshifts.
+    CORR_LX : float, optional
+        Self-correlation length used for updating halo properties. Properties are interpolated between
+        a random sample at the current halo mass and one matching the point in the PDF of the
+        previous sample, the interpolation point in [0,1] is given by exp(-dz/CORR_STAR)
```

Comment:
does this mean to say `CORR_LX` instead of `CORR_STAR`? 

I *think* I understand this explanation, but I'm not quite sure: which properties are interpolated? All of them? "Previous" means higher redshift?

Reply:
OK, that is good -- however I still feel like it should be `CORR_LX` instead of `CORR_STAR`? You're describing the parameter `CORR_LX`.

Reply:
Ah, I see all is good, sorry.